ament_package/license.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class License:
+    __slots__ = ['file']
+
+    def __init__(self, name, *, file=None):
```

Comment:
Please use `file_` for the argument - same as in [catkin_pkg](https://github.com/ros-infrastructure/catkin_pkg/pull/202/files#diff-376c1537291f451949ade47ac2a5af34R376).

Reply:
Done.
I separated the commit because I felt it a bit uncommon adding an underscore to keyword argument. (I didn't notice it for ``catkin_pkg`` because I treated it as a [positional argument](https://github.com/ros-infrastructure/catkin_pkg/pull/202/files#diff-376c1537291f451949ade47ac2a5af34R567) .)

Anyway, I'll leave it to you.

Reply:
I agree that it is not "pretty" - just following the Python naming guidelines...